### PR TITLE
report: correct audit titles that should not be capitalized

### PR DESCRIPTION
**Pull request by connorjclark**
Couldn't unsee once I noticed, so here's some trivial copy changes.

Before:
image

After:
image

### CODE REVIEW

File: lighthouse-core/audits/critical-request-chains.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const ComputedChains = require('../computed/critical-request-chains.js');
 
 const UIStrings = {
   /** Imperative title of a Lighthouse audit that tells the user to reduce the depth of critical network requests to enhance initial load of a page. Critical request chains are series of dependent network requests that are important for page rendering. For example, here's a 4-request-deep chain: The biglogo.jpg image is required, but is requested via the styles.css style code, which is requested by the initialize.js javascript, which is requested by the page's HTML. This is displayed in a list of audit titles that Lighthouse generates. */
-  title: 'Minimize Critical Requests Depth',
+  title: 'Minimize critical requests depth',
```
Comment (patrickhulce) [inline lighthouse-core/audits/critical-request-chains.js]: Now that I'm looking at it like an imperative sentence, I'm not sure this is right...
```suggestion
  title: 'Minimize depth of critical requests',
```
or
```suggestion
  title: 'Avoid chaining critical requests',
```

perhaps?

Comment (connorjclark) [reply lighthouse-core/audits/critical-request-chains.js]: I think "chaining" conveys what's going on better.

File: lighthouse-core/audits/third-party-summary.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ const MainThreadTasks = require('../computed/main-thread-tasks.js');
 
 const UIStrings = {
   /** Title of a Lighthouse audit that identifies the code on the page that the user doesn't control. This is shown in a list of audits that Lighthouse generates. */
-  title: 'Third-Party Usage',
+  title: 'Third-Party usage',
```
[the remark this answers is not in the document]
Comment (paulirish) [reply lighthouse-core/audits/third-party-summary.js]: +1 to adding a verb. maybe `Minimize third-party usage`?

**Comment by paulirish** [comment]
@meggynw  can you review these changes? wanted to make sure they are reasonable to you.

File: lighthouse-core/audits/critical-request-chains.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ const ComputedChains = require('../computed/critical-request-chains.js');
 
 const UIStrings = {
   /** Imperative title of a Lighthouse audit that tells the user to reduce the depth of critical network requests to enhance initial load of a page. Critical request chains are series of dependent network requests that are important for page rendering. For example, here's a 4-request-deep chain: The biglogo.jpg image is required, but is requested via the styles.css style code, which is requested by the initialize.js javascript, which is requested by the page's HTML. This is displayed in a list of audit titles that Lighthouse generates. */
-  title: 'Minimize Critical Requests Depth',
+  title: 'Avoid chaining critical requests',
```
Comment (paulirish) [inline lighthouse-core/audits/critical-request-chains.js]: yah this is an improvement in clarity

File: lighthouse-core/lib/i18n/en-US.json [hunk 4]
```diff
@@ -0,0 +1,1858 @@
+{
+  "lighthouse-core/audits/accessibility/accesskeys.js | description": {
```
Comment (patrickhulce) [inline lighthouse-core/lib/i18n/en-US.json]: this seems like a mistaken file to commit or bad merge or something @connorjclark ?

Comment (brendankenny) [reply lighthouse-core/lib/i18n/en-US.json]: > this seems like a mistaken file to commit or bad merge or something @connorjclark ?

fixed

File: lighthouse-core/audits/third-party-summary.js [hunk 5]
```diff
@@ -15,7 +15,7 @@ const MainThreadTasks = require('../computed/main-thread-tasks.js');
 
 const UIStrings = {
   /** Title of a diagnostic audit that provides details about the code on a web page that the user doesn't control (referred to as "third-party code"). This descriptive title is shown to users when the amount is acceptable and no user action is required. */
-  title: 'Third-Party usage',
+  title: 'Limit third-party usage',
```
Comment (brendankenny) [inline lighthouse-core/audits/third-party-summary.js]: only remaining thing is @paulirish suggested `Minimize third-party usage` as an alternative verb here

**Pull request merged**